express-app/app.models.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../db/connection');
+
+exports.fetchTopics = (slug, description) => {
+  let query = `SELECT * FROM topics`;
+  if (slug) {
+    query += ` WHERE slug = ${'slug'}`;
+  }
+  if (description) {
+    query += ` WHERE description = ${`description`}`;
+  }
```

Comment:
I like the idea of making this function dynamic, but following TDD we should only have code that has been tested for, so this shouldnt be included in this pr